### PR TITLE
Homework/2

### PR DESCRIPTION
**Pull request by canzoroglu**

### CODE REVIEW

File: Homework2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        {
+            int[] copiedArray = new int[arr.Length + 1];
+            Array.Copy(arr, copiedArray, arr.Length);
+            copiedArray[copiedArray.Length - 1] = item;
+            return copiedArray;
+        }
+
+        static int[] AddToArray(int[] arr, int item, int index)
+        {
+            if (index == 0) return AddToStart(arr, item);
```
Comment (westonpace) [inline Homework2/Program.cs]: This is backwards.  There is not much advantage to deferring to `AddToStart`/`AddToEnd`.  If you wanted to simplify things you could do the reverse.  `AddToStart` could be `AddToArray(arr, item, 0)` and `AddToEnd` could be `AddToArray(arr, item, arr.Length)`

File: Homework2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            else if (index == arr.Length - 1) return RemoveFromEnd(arr);
+            int[] copiedArray = new int[arr.Length - 1];
+            Array.Copy(arr, copiedArray, index);
+            Array.Copy(arr, index + 1, copiedArray, index, copiedArray.Length - index);
+            return copiedArray;
+        }
+
+        static void SignUp()
+        {
+            Console.Write("Add an username:");
```
Comment (westonpace) [inline Homework2/Program.cs]: Technically "a username".  I wouldn't pick on this normally but I'm finding very little wrong so I have to nitpick where I can :)

**Review by westonpace — COMMENTED** [review]
Very well done.  I don't have much feedback because it is very solid.

**Comment by canzoroglu** [comment]
Thank you very much @westonpace for your nice comments and review :) I've made the changes you mentioned.

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**